packages/mds-ingest-service/repository/entities/event-entity.ts

```diff
@@ -28,6 +30,7 @@ export interface EventEntityModel extends IdentityColumn, RecordedColumn {
   telemetry_timestamp: EventDomainModel['telemetry_timestamp']
   trip_id: EventDomainModel['trip_id']
   service_area_id: EventDomainModel['service_area_id']
+  telemetry?: Nullable<TelemetryEntityModel>
```

Comment:
Regarding the other entity properties. We've recently stopped defining them by referencing the domain model (even when equivalent) as it causes unnecessary coupling of the models and also creates some issues for certain reflection operations. Also, we've been defining the entity itself without an `implements` clause and then just `export type EventEntityModel = TripEntity` for consistency.

Reply:
I was trying to figure out what I did wrong here! But then I realized you're asking me to refactor the models from this repo because you were the last commiter? :)

Reply:
This is a good idea, but I'm going to let another PR address this refactoring.